src/the-bureau/_macro-role.html

```diff
@@ -0,0 +1,33 @@
+{# ==========================================================================
+
+   role.render()
+
+   ==========================================================================
+
+   Description:
+
+   Renders a time element with a standard datetime format when given:
```

Comment:
Argh! Nope. I accidentally had the right code comment in the file I needed to delete and not vice versa. Fixed in https://github.com/cfpb/cfgov-refresh/commit/877d7e3e0a9e78c0c4cee90b6e20c90949b1dc1e